src/components/Header/Header.scss

```diff
@@ -0,0 +1,5 @@
+.header {
+	display: flex;
+	justify-content: center;
+	margin-bottom: 30px;
```

Comment:
should use rem
https://uxdesign.cc/why-designers-should-move-from-px-to-rem-and-how-to-do-that-in-figma-c0ea23e07a15#:~:text=The%20beauty%20of%20it%20is,needs%20to%20adapt%20font%20sizes.